runfullday.sh

```diff
@@ -192,22 +198,27 @@ fi
 
 #
 # Check the 256 byte, 10 Flow test, and make sure they have at least 75%
-# of 10G line rate at L3 [3396739,1325pps]
+# of line rate at L3
```

Comment:
Can we keep it 75% of 10G as the performance here is based on the Cores assigned to DPDK, not the actual speed of the NIC. This does make sense in the TC Flower case.

Reply:
Ok, done with V3